code/game/objects/items/devices/radio/radio.dm

```diff
@@ -240,7 +240,8 @@
 	var/turf/position = get_turf(src)
 
 	var/jammed = FALSE
-	for(var/obj/item/device/jammer/jammer in GLOB.active_jammers)
+	for(var/i in GLOB.active_jammers)
+		var/datum/component/jammer/jammer = i
 		if(get_dist(position,get_turf(jammer)) < jammer.range)

```

Comment:
get_turf() on a datum?

Reply:
woops, if this doesn't work I can always make it parent.get_turf...